brownie/network/contract.py

```diff
@@ -530,6 +684,151 @@ def balance(self) -> Wei:
         balance = web3.eth.getBalance(self.address)
         return Wei(balance)
 
+    def publish_source(self, silent: bool = False) -> bool:
```

Comment:
Same comment as above...  this is probably better as `ContractContainer.publish_source(contract)` where you hand it a `Contract` object.  Otherwise more collisions :scream: